test/test_dependencies.jl

```diff
@@ -0,0 +1,54 @@
+using Mimi
+
+Pkg.add("InfoZIP")
+
+#list of URLs of branches of packages to test
+dependencies = [
+"https://github.com/davidanthoff/fund.jl/archive/master.zip"
+]
+
+#list of failed tests to build as you go
+errors = []
+#make a temporary directory to run the tests in
+tmp_path = joinpath(dirname(@__FILE__),"tmp_testing/")
+mkdir(tmp_path)
+cd(tmp_path)
+
+#loop through each dependent package
+for url in dependencies
+  #download the package
+  download(url)
+  zip_name = chomp(basename(url))
```

Comment:
Does ``basename`` work here? I thought it it meant for filenames, not URLs? I might be wrong.